news/tasks/extraction.py

```
@@ -0,0 +1,87 @@
+    2. DB 중복 체크 (이미 저장된 기사는 건너뜀)
+    3. Jina.ai로 본문 추출
+    4. 추출 결과 검증 및 반환
+
+    Returns:
+        raw_content가 추가된 기사 딕셔너리 (실패 시 None)
+    """
+    url = article.get("link")
+
+    # URL 유효성 검증
+    if not url:
+        logger.warning(f"[Extract] URL 없음: 건너뜀")
```

Ruff (0.14.10)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)